prompting/task_registry.py

```diff
@@ -0,0 +1,9 @@
+# TODO: Expand this to include extra information beyond just the task and dataset names
+TASK_REGISTRY = {
+    "mock": ["mock"],
```

Comment:
might make more sense to define these nicknames in the classes themselves. We can add an alias or nickname attribute each dataset class and then just reference it here eg.
```python
wiki = WikiDataset.nickname
math = MathDataset.nickname
...

TASK_REGISTRY = {
    'summarization': [wiki],
    'qa': [wiki],
}
```
same should also go for the task nicknames. We shouldn't rely on remembering their nicknames.

Reply:
Yup that's an easy change to make